[14.0][FIX] l10n_es_aeat_sii_oca: SII Refund Type not required

joao-p-marques opened the pull request:
FWP of https://github.com/OCA/l10n-spain/pull/1722

Make the SII Refund Type not required nor available for account.move records that are not "in_invoices" or "out_invoices"

@Tecnativa
TT30339

ping @pedrobaeza

pedrobaeza commented on l10n_es_aeat_sii_oca/wizards/account_move_reversal.py:
```diff
@@ -12,7 +12,14 @@ def _default_sii_refund_type_required(self):
+        # show the field
+        return bool(
+            invoices.filtered(
+                lambda i: i.type in ("in_invoice", "out_invoice")
```
In v14 it's `move_type`

joao-p-marques replied:
Sorry again :man_facepalming: 

Changes made :+1:

joao-p-marques committed:
[FIX] l10n_es_aeat_sii_oca: SII Refund Type not required

Make the SII Refund Type not required nor available for account.move records that are not "in_invoices" or "out_invoices"

TT30339

pedrobaeza commented:
/ocabot merge patch

HaraldPanten reviewed (approved):
Igual que en V13. LGTM

pedrobaeza commented:
/ocabot merge patch
